redash/models/__init__.py

```diff
@@ -331,6 +331,7 @@ class QueryResult(db.Model, QueryResultPersistence, BelongsToOrgMixin):
     _data = Column("data", db.Text)
     runtime = Column(postgresql.DOUBLE_PRECISION)
     retrieved_at = Column(db.DateTime(True))
+    db_role = Column(db.String(128), nullable=True)
```

Comment:
what happens when it's None? dont set role?

Reply:
ah this is on query results, so users will only be able to see their own queries then?